misc editorial

jmdyck opened the pull request:
(Assuming I follow your thinking.)

jmdyck commented:
Rebased to your revised fork.

jmdyck commented:
Any reason not to pull these commits, @littledan?

littledan commented:
Hey, thanks for the ping, just haven't gotten around to reviewing them yet, sorry for the delay.

jmdyck commented:
Okay, no problem.

jmdyck commented:
Hm, I'm having trouble resolving the conflicts.

littledan commented:
Not sure what you mean. I've (tried to) pulled in your changes. See https://github.com/tc39/ecma262/pull/1135/commits . What conflicts are you working on resolving?

jmdyck commented:
Actually, I got conflicts just trying to rebase my fork of number-refactoring. But it looks like you got my changes, so I think I'll blow that away and start fresh.

littledan commented:
@jmdyck Thanks for all your great work here.

jmdyck committed:
Editorial: "Number value of" -> "Number value for"

Conversion of mathematical values to Number values
is already described in "The Number Type",
and is denoted by the phrase "the Number value for X".

jmdyck committed:
Editorial: insert "the Number value for" in a few spots

(no implicit conversions!)

jmdyck committed:
Editorial: reword "the number whose value is MV of |NumericLiteral|"

The phrase:
    "the number whose value is MV of |NumericLiteral|"
could be interpreted as:
    "the Number value for MV of |NumericLiteral|"
but this would be incorrect, because
"the Number value for" (6.1.6) doesn't completely
describe the rounding that is applied to the MV
when obtaining the Number represented by |NumericLiteral|.

Instead, change to:
    "the Number value represented by |NumericLiteral|"
which leaves MV out of it, and so refers to 11.8.3's
whole process for arriving at a Number value.
